docs/commands.md

```diff
@@ -46,6 +46,10 @@ Uses a custom directory instead of `<projectName>`.
 
 Uses a custom app title instead of `<projectName>`.
 
+#### `--template-version <string>
+
+Use a specific version of the [Community Template](https://github.com/react-native-community/template).  This is a shortcut for `--template @react-native-community/template@<version>`. It will default to the latest version if the `--template-version` argument isn't provided.
```

Comment:
Do we expect users to use this? It seems unnecessary at first, but as we have it right now it's ok to keep it

Reply:
It can be stripped because we can achieve the same effect with `--template`.

Reply:
If possible, I won't expose it at first. It's yet another flag/api to maintain. We can still expose it if a user needs it afterwards